configs/amcts/gatekeeper.cfg

```diff
@@ -0,0 +1,8 @@
+@include ../../engines/KataGo-raw/cpp/configs/training/gatekeeper1.cfg
+@include ../active-experiment.cfg
```

Comment:
Why did you choose to `@include` this rather than add is an extra config via the `-config` parameter? Naively the latter would give some more flexibility if we want to use a different config rather than just mutating active-experiment.cfg. But perhaps we need to override things in a specific order, for example?

Reply:
I don't think I actually thought about this when implementing it. But in this case I think the override order actually does matter —
* We expect to include `active-experiment.cfg` after `gatekeeper1.cfg` because we want to override `gatekeeper1.cfg` defaults with our `active-experiment.cfg` setup
* But we also need to define `noResultStdev = 0.0` and `drawRandRadius = 0.0` or else the gatekeeper will complain and exit. (`active-experiment.cfg` imports `victimplay.cfg`, which has `noResultStdev` and `drawRandRadius` as non-zero values